### PR TITLE
Feature/regra ativação simplificada

### PR DESCRIPTION
**Pull request by anamaiabap**
1. Pegando os valores validos para os campos de Produto, Sku, Categoria, Marca, Coleção, Especificação
2. Criando as queries para o 1.
3. Todos os campos de preenchimento de regra de ativação devem ser input
4. Criando o botão de salvar as modificações

### CODE REVIEW

File: node/clients/product.ts [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      const value = await this.http.get(
+        `https://${workspace}.vtexcommercestable.com.br/api/catalog_system/pub/specification/field/listByCategoryId/${id}`
+      )
+
+      value.forEach((element: { FieldId: number; Name: string }) => {
+        const type = element.Name
+
+        names.push(`${type}`)
+      })
+    }
```
Comment (gabrielHosino) [inline node/clients/product.ts:126]: I think you can do something similar to what i suggested on getProductsNames

File: node/clients/sku.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    const names: string[] = []
+
+    for (const id of ids) {
+      // eslint-disable-next-line no-await-in-loop
+      const value = await this.http.get(
+        `https://${workspace}.vtexcommercestable.com.br/api/catalog_system/pvt/sku/stockkeepingunitbyid/${id}`
+      )
+
+      names.push(value.NameComplete)
+    }
```
Comment (gabrielHosino) [inline node/clients/sku.ts:49]: Here too 😄

File: react/Provider/provider.tsx [hunk 3]
```diff
@@ -13,6 +14,10 @@ const Provider: FC = (props) => {
     setFile({ ...file, ...{ result: files } })
   }
 
+  function save() {
+    // inserir regra
+  }
```
Comment (gabrielHosino) [inline react/Provider/provider.tsx:19]: Is this a TODO?

File: react/utils/functionsConditions.tsx [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+  return (
+    <Dropdown
+      value={value}
+      options={options}
+      onChange={(e: { target: { value: any } }) => {
+        onChange(e.target.value)
+      }}
+    />
+  )
+} */
```
Comment (gabrielHosino) [inline react/utils/functionsConditions.tsx:34]: Will this be used later?

File: admin/routes.json [hunk 5]
```diff
@@ -2,9 +2,5 @@
   "admin.app.example": {
     "component": "AdminExample",
     "path": "/admin/app/badges"
```
Comment (brunorodmoreira) [inline admin/routes.json:4]: it would be good to change these boilerplate names already

File: graphql/schema.graphql [hunk 6]
```diff
@@ -0,0 +1,9 @@
+type Query {
+  getProductsIds: [Int]
+  getProductsNames: [String]
+  getSkuNames: [String]
+  getBrandsNames: [String]
+  getCollectionsNames: [String]
+  getCategoryName: [String]
+  getSpecificationName: [String]
+}
```
Comment (brunorodmoreira) [inline graphql/schema.graphql:9]: it would also be good to remove this query if it's not being currently used.

File: manifest.json [hunk 7]
```diff
@@ -15,7 +15,53 @@
     "react": "3.x",
     "admin": "0.x",
     "messages": "1.x",
-    "docs": "0.x"
+    "docs": "0.x",
+    "graphql": "1.x",
+    "node": "6.x"
```
Comment (brunorodmoreira) [inline manifest.json:20]: Same as before, removing unused builders.

```suggestion
    "docs": "0.x"
```

File: manifest.json [hunk 8]
```diff
@@ -15,7 +15,53 @@
...
+      }
+    },
+    {
+      "name": "outbound-access",
+      "attrs": {
+        "host": "{{workspace}}.vtexcommercestable.com.br",
+        "path": "/api/*"
+      }
+    }
+  ]
```
Comment (brunorodmoreira) [inline manifest.json:66]: Removing unnecessary policies.

```suggestion
  "$schema": "https://raw.githubusercontent.com/vtex/node-vtex-api/master/gen/manifest.schema"
```

File: messages/pt.json [hunk 9]
```diff
@@ -1,4 +1,4 @@
 {
-  "badges.navigation.label": "Gerenciamento de Badges",
+  "badges.navigation.label": "Gerenciamento de Badges Novo",
```
Comment (brunorodmoreira) [inline messages/pt.json:2]: It would be nice to commit with a proper message here or with a marked `TODO`

**Review by brunorodmoreira — CHANGES_REQUESTED** [review]
@anamaiabap I think you should remove that unused structured before submit to merging. You can use the [stash](https://medium.com/wooza/git-stash-conhecendo-e-utilizando-um-dos-comandos-mais-pr%C3%A1ticos-para-o-versionamento-de-seu-c%C3%B3digo-a4dab3ac70da) commando from git

File: react/Provider/provider.tsx (hunk 3 above)
Comment (anamaiabap) [reply react/Provider/provider.tsx:19]: Exato, vai estar no proximo PR :)

**Pull request merged**